packages/react-styles/src/css/components/Topology/topology-pipelines.css

```diff
@@ -0,0 +1,352 @@
+
+  --pf-topology-pipelines__pill--Color: var(--pf-global--Color--100);
+
+  /* dark node */
+  --pf-topology-pipelines__pill__background--Fill: var(--pf-global--BackgroundColor--300);
+  --pf-topology-pipelines__pill__background--Stroke:  var(--pf-global--palette--black-300);
+
+  --pf-topology-pipelines__pill--m-disabled--Background--Fill: var(--pf-topology-pipelines__pill__background--Fill);
+  --pf-topology-pipelines__pill--m-disabled--Background--Stroke: var(--pf-global--palette--black-500);
+
+  --pf-topology-pipelines__pill--m-dragging--background--StrokeWidth: 2px;
+
+  --pf-topology-pipelines__pill--m-hover--background--Stroke: var(--pf-global--palette--black-500);
+  --pf-topology-pipelines__pill--m-hover--label__background--Stroke: var(--pf-global--palette--black-100);
+
+  /* dark node labels */
+  --pf-topology-pipelines__pill__label__background--Fill: var(--pf-global--BackgroundColor--300);
+  --pf-topology-pipelines__pill__label__background--Stroke: var(--pf-global--BorderColor--300);
+  --pf-topology-pipelines__pill__label--m-dragging--background--StrokeWidth: 2px;
+  --pf-topology-pipelines__pill--text-Color: var(--pf-global--Color--100);
+
+  /* dark selected nodes */
+  --pf-topology-pipelines__pill--m-selected__background--Stroke: var(--pf-global--palette--blue-300);
+  --pf-topology-pipelines__pill--m-selected__label__background--Stroke: var(--pf-global--palette--blue-400);
+  --pf-topology-pipelines__pill--m-selected__label__background--Fill: var(--pf-global--palette--blue-400);
+  --pf-topology-pipelines__pill--m-selected--separator--Stroke: var(--pf-global--palette--blue-300);
+
+  --pf-topology-pipelines__pill--m-selected--text-Color: var(--pf-global--Color--100);
+  --pf-topology-pipelines__pill--m-selected--m-info--text-Color: var(--pf-global--palette--black-900);
+  --pf-topology-pipelines__pill--m-selected--m-danger--text-Color: var(--pf-global--palette--black-900);
+  --pf-topology-pipelines__pill--m-selected--m-warning--text-Color: var(--pf-global--palette--black-900);
+  --pf-topology-pipelines__pill--m-selected--m-success--text-Color: var(--pf-global--palette--black-900);
+
+  --pf-topology-pipelines__pill__label__icon--Color: var(--pf-global--palette--black-600);
+
+  --pf-topology-pipelines__pill__action-icon__icon--Color: var(--pf-global--Color--100);
+}
+
+/* Color is defined in the svg but fill will override it */
+.pf-topology-pipelines__pill {
+  fill: var(--pf-topology-pipelines__pill--Color);
+}
+
+.pf-topology-pipelines__pill.pf-m-hover {
+  --pf-topology-pipelines__pill__background--StrokeWidth: 2px;
+}
+
+.pf-topology-pipelines__pill--background {
+  fill: var(--pf-topology-pipelines__pill__background--Fill);
+  stroke-width: var(--pf-topology-pipelines__pill__background--StrokeWidth);
+  stroke: var(--pf-topology-pipelines__pill__background--Stroke);
+}
+
+.pf-topology-pipelines__pill--background.pf-m-disabled {
```

Comment:
`pf-topology-pipelines__pill__background` to match the class names
... and the classes below, too.